lfs/config.go

```diff
@@ -544,6 +565,38 @@ func (c *Configuration) readGitConfig(output string, uniqRemotes map[string]bool
 	}
 }
 
+// Special handling for url.<url>.insteadOf, since <url> will contain '.'
+func (c *Configuration) readGitInsteadOfConfig(output string, uniqInsteadOfs map[string]string) {
+	lines := strings.Split(output, "\n")
+	uniqKeys := make(map[string]string)
+	for _, line := range lines {
+		// Why is this not "="?
+		pieces := strings.SplitN(line, " ", 2)
```

Comment:
> // Why is this not "="?

This is the first time the `pieces` var is used in this scope, so it must be declared. Same goes for values like `key` or `value` below. I suppose we could declare them above the `for` block, but I don't think it's worth the hassle.